File: arrays/maxConsecutiveOnes.cpp
class Solution {
public:
    int findMaxConsecutiveOnes(vector<int>& nums) {
        int global = 0;
        int local = 0;
        for(int i =0; i<nums.size();i++){
            if(nums[i]==1){
                local +=1;
                if(global<local)global = local;
            }else{
                local = 0 ;
            }
        }
        return global;
    }
};

File: arrays/maxConsecutiveOnes.py
class Solution(object):
    def findMaxConsecutiveOnes(self, nums):
        m = 0
        c = 0
        for i in nums:
            if i == 1:
                c += 1
            else:
                m = max(m, c)
                c = 0
        return max(c, m)
        